### ISSUE TITLE
react hot loader 4.6 broken

### ISSUE THREAD

**Issue by pungggi** [solkimicreb/react-easy-state#77]
Hello first of all, great project!
I am not sure where to start explaining my issue maybe its here may be its react hot loader..
maybe its linaria.

I am using gatsby with react easy state.
gatsby 2.0.69 introduced a new version of react-hot -loader 4.6

Then I use a css in js solution called [linaria](https://github.com/callstack/linaria).

As of this version Props passed to a  styled` component are not hot updated  **if they are wrapped with view from react easy state**..

For example:

Box.js
```js
import { styled } from "linaria/react"

const Box = styled.div`
  color: ${(props) => (props.color ? props.color : "red")};

export default Box
```

then i consume

index.js

```js
import { view } from 'react-easy-state'

export default view(() => {
  const classColor = css`
    color: green;
  `
  return (
     <Box padding={[30, 2, 10, 20]}>
       <div className={classColor}>
         <h1>About</h1>
       </div>
     </Box>
  )
})
```
If I change the classColor it updates.
But changing the value of the `padding` prop does not trigger a reload. This was working with gatsby 2.0.68 that uses react hot loader 4.5.

**Comment by solkimicreb** [comment]
Which version of react-easy-state are you using? v6.1.x migrated to React hooks when possible under the hood. Maybe the issue has to do something with that.

**Comment by pungggi** [comment]
I does not matter if 6.1.1 or 6.0.6 both behave the same in this case

**Comment by pungggi** [comment]
I setup a repo for reproduction 

https://github.com/pungggi/rhl

see Branch hotload4_6 this uses the latest packages with react hot loader ^4.6.0

**Comment by solkimicreb** [comment]
Thanks! I will look into this.

**Comment by pungggi** [comment]
opened an [issue ](https://github.com/gaearon/react-hot-loader/issues/1192) in react hot loader too.

**Comment by solkimicreb** [comment]
@pungggi Did you manage to solve this?

**Comment by buzinas** [comment]
@pungggi why was this issue closed? How can I make react-hot-loader work with react-easy-state?

**Comment by n3v3rd1e** [comment]
Hello, did anyone managed to solve this, or is it still not working?

**Comment by n3v3rd1e** [comment]
I do not mean to spam, but I feel like this is a really important issue and should not be ignored as it makes easy-state kinda not usable right now. Some statement about this would be highly appreciated, thank you.

**Comment by solkimicreb** [comment]
Looking into this. It seems to be an issue with functional components only. Did anyone have issues with class components and HMR?

**Comment by solkimicreb** [comment]
I just released v6.1.3 which hopefully fixes this issue 🎉 Please reopen it if the issue stays for any of you. Sorry for the long wait!

**Comment by n3v3rd1e** [comment]
Nice work perfectly, thanks 👍
